hashtables/ex1/ex1.py

```diff
@@ -12,12 +10,21 @@ def get_indices_of_item_weights(weights, length, limit):
     """
     YOUR CODE HERE
     """
+    
+    for i in range(length):
+        hash_table_insert(ht, weights[i], i)
+
+    for i in range(length):
```

Comment:
small optimization - consider adding the functionality of the second for loop to the first for loop.  that is, we can check for compliments in the hash table as we are adding to it as opposed to waiting until the entire thing is filled first.